scripts/mkcloud

```diff
@@ -132,9 +132,18 @@ if [[ $want_magnum_proposal = 1 ]]; then
     : ${compute_node_memory:=6621440}
     : ${computenode_hdd_size:=80}
 # octavia requires 1GB of RAM for every load balancer it creates, so
-# it needs compute nodes to be a little beefier
+# it needs compute nodes to be a little beefier in evironments with two or
+# fewer compute nodes
 elif [[ $want_octavia_proposal = 1 ]]; then
-    : ${compute_node_memory:=3145728}
+    compute_nodes=$(( $(nodes number normal) - 1 ))
```

Comment:
Hm, don't we have better way of figuring out the number of compute nodes?

Reply:
i sugges to either do a knife search like in `select_compute_nodes` or parse the proposal items like in `get_novacontroller`

Reply:
@jsuchome yes, I asked myself the same question. I even tried using the `qa_crowbarsetup` functions for this, but I realized this logic needs to run long _before_ the mkcloud environment is up and running. That means I'm not able to use `knife` or any other Crowbar command here, so I'm stuck with this rudimentary logic I'm not too happy about either.

Reply:
On the upside, I only enabled Octavia in a handful of mkcloud jobs, so this logic doesn't need to cover a wider range of use-cases.

Reply:
Hm, true, you can't use my suggestions.
What about passing `compute_node_memory` as a variable from outside, together with `want_octavia_proposal`? That way it would be clearly from the place where the number of nodes is defined.